drafts/securing-openliberty-with-openidconnect-and-jwt.adoc

```diff
@@ -11,49 +11,42 @@ blog_description: "A look at the OpenID Connect, SAML and Json Web Token securit
 = Securing Open Liberty apps and microservices with OpenID Connect and JWT
 Bruce Tiffany <https://github.com/brutif>
 
-Open Liberty now includes more features for securing apps with external security providers.  
-Let's take a quick look at why and how to use some of these features, focusing on using OpenID Connect to secure microservices.
+Open Liberty now includes more open source security features to enable you to secure your apps with external security providers. Let's take a quick look at why and how to use some of these features.
 
-== Concepts:
-* SAML and OpenID Connect are standards-based technologies for using an external security provider to secure your applications. 
-+
-* Json Web Tokens (JWT) are identity credentials with a standardized format that can be exchanged between processes. They're secured against tampering by use of digital signatures.
+== Authenticating microservices on Open Liberty
+Open Liberty provides support for writing apps that authenticate users with social media logins using Open ID standards-based technologies. You can present users of your web app with the options to log in using their existing Facebook, GitHub, Google, LinkedIn, or Twitter IDs. You can also write your own integrations with other social media platforms if you need to.  Alternatively, you can enable your app users to use single sign-on by authenticating with a SAML identity provider.
 
-== Advantages of using OpenID Connect or SAML to secure apps:
-* User accounts are managed by the OpenID Connect or SAML provider, relieving administrators and developers of this effort.
-+
-* The secured app never sees the user's password.
-+
-* A user of multiple apps secured by the same provider can have a "single sign on" experience where they only need to log in once for all the secured apps.
-+
-* OpenID Connect providers can issue Json Web Token (JWT) credentials.  These tokens can be submitted to downstream secured microservices to propagate identity in an efficient, self-contained way.  Microservices don't need to contact the identity provider again.
+Using Open ID Connect or SAML providers ensures that the secured app never sees the user's password, and relieves developers and administrators of the effort of managing user accounts. In addition, providing a single sign-on experience to users enables them to log in once for all secured apps that they use.
+
+Open Liberty also supports (JWT), which are identity credentials with a standardized format that can be exchanged between processes. They're secured against tampering by use of digital signatures. So your microservices can securely exchange requested information on behalf of users. OpenID Connect providers can issue Json Web Token (JWT) credentials.  These tokens can be submitted to downstream secured microservices to propagate identity in an efficient, self-contained way.  Microservices don't need to contact the identity provider again.
```

Comment:
add Json Web Tokens or remove () around JWT. It's not really correct that they allow microservices to securely exchange information - they're more narrow - they only tell the recipient who is calling them. Thining about this one too.